driver-core/src/main/java/com/datastax/driver/core/Metrics.java

```diff
@@ -273,6 +273,7 @@ void shutdown() {
         private final Counter writeTimeouts = registry.counter("write-timeouts");
```

Comment:
ah, maybe this is not a problem since Errors is a non-static nested class, so couldn't be extended anyways.